src/components/feature-board-external-tool-element/ExternalToolElementAlert.vue

```diff
@@ -25,31 +27,44 @@ export default defineComponent({
 		error: {
 			type: Object as PropType<BusinessError>,
 		},
-		isToolOutdated: {
-			type: Boolean,
+		toolOutdatedStatus: {
+			type: Object as PropType<ToolConfigurationStatus | undefined>,
+			required: true,
```

Comment:
this would be the correct way of doing properties that can be undefined, since undefined is not an Object
```suggestion
		toolOutdatedStatus: {
			type: Object as PropType<ToolConfigurationStatus>,
```